packages/@okta/vuepress-site/docs/guides/add-an-external-idp/configure-idp-in-okta/oktatookta/appidpinokta.md

```diff
@@ -0,0 +1,11 @@
+By default, Okta requires the `email` attribute for a user. The `email` scope is required to create and link the user to Okta's Universal Directory. If your Identity Provider doesn't support this attribute, you can make it optional. See [Manage User Profiles](https://help.okta.com/en/prod/Content/Topics/Directory/eu-profile-editor.htm).
+
+After you fill in your OpenID Connect Identity Provider's settings, add the following endpoint URIs for the OpenID Connect Identity Provider that you are configuring. You can find the endpoints in the well-known configuration document for the Identity Provider (for example, `https://theIdPorg.com/.well-known/openid-configuration`. 
```

Comment:
For Okta-to-Okta, can we be more specific? The developer is using Okta as an IdP, not a generic `OpenID Connect Identity Provider`.

Reply:
We should format readably (`Okta-to-Okta`), but I think it'd be helpful to clarify that this is Okta-to-Okta, not just Okta.

"Using Okta as an identity provider", conceptually, usually means using your Okta org as your identity provider. It might be helpful to clarify that here this means using a _separate_ Okta org as your provider.

We should probably have conceptual documentation on this eventually as well, but developers may encounter this here without any background.

Reply:
@alexdahl-okta - there is conceptual information on this. I will add a link in this section to it, also. https://developer.okta.com/docs/concepts/identity-providers/

Reply:
I would also recommend that we reformat `OPENIDCONNECT.` Can we add spaces? If not `OpenID-Connect`?

Reply:
Is there a reason for the order `okta-to-okta` `openidconnect` `saml`? I would recommend that we list them based on how likely a developer is going to configure the type of application.

Reply:
The system automatically puts them in alphabetical order.

Reply:
> I would also recommend that we reformat `OPENIDCONNECT.` Can we add spaces? Do all letters have to be uppercase? If not, `OpenIDConnect` or `OpenID-Connect`?

@ZoeMonosson-okta - Alex has already fixed this with his PR from this morning. Thanks!